template/cordova/lib/JsprojManager.js

```diff
@@ -121,7 +121,12 @@ jsprojManager.prototype = {
         copyToOutputDirectory.text = 'Always';
         children.push(copyToOutputDirectory);
 
-        var item = createItemGroupElement('ItemGroup/Content', sourcePath, targetConditions, children);
+        var item;
```

Comment:
IMO it'd be better to calculate path for `content` element outside this method (in `'resource-file'.install`) - this way you will avoid checking the same condition (`if (targetConditions.reference) ...`) twice and won't need to modify this method at all.